### ISSUE TITLE
Some little refusals in it.json

### ISSUE THREAD

**Issue by davideriboli** [happydata/nomie-languages#4]
Hi @happydata and @landreoli,

Nomie is a great app and it was a pleasure to find out that someone was working on an Italian location. I've found some refusals here and there and I hope this review will be useful. In general I tried to:

1) correct some small typos like "sider" instead of "sliders";
2) normalize the use of capital letters according to Italian grammar,
3) to normalize the use of plurals of foreign terms according to Italian grammar.

[it.json.zip](https://github.com/happydata/nomie-languages/files/946153/it.json.zip)

```
{
    "NOMIE_ACCOUNT": "Account Nomie",
    "VIEW_DAY": "Visualizza giorno",
    "DEFAULT_BOARD_NAME": "Principale",
    "SORT": "Ordina",
    "DELETE": "Rimuovi",
    "RENAME": "Rinomina",
    "LANGUAGE" : "Lingua",
    "TYPE_SINGLE_TAP": "Singolo Clic",
    "TYPE_NUMERIC": "Numero",
    "TYPE_RANGE": "Slider",
    "TYPE_TIMER": "Timer",
    "TYPE_BOOLEAN": "ON / OFF",
    "TYPE_SINGLE_TAP_DESCRIPTION": "Clicca per aggiungere",
    "TYPE_NUMERIC_DESCRIPTION": "Inserisci valore numerico",
    "TYPE_RANGE_DESCRIPTION": "Seleziona in un intervallo",
    "TYPE_TIMER_DESCRIPTION": "Clicca per iniziare, clicca per terminare",
    "TYPE_BOOLEAN_DESCRIPTION": "Interruttore giornaliero",
    "MY_CLOUD_APPS": "Mie Applicazioni Cloud",
    "HOW_ARE_YOU_DOING?": "Come stai?",
    "BROWSE": "Sfoglia",
    "CLOUD_APP_SUMMARY": "Le Applicazioni Cloud sono servizi web esterni che possono monitorare, analizzare e comunicare informazioni su tracker specifici scelti da te.",
    "AUTO_SORT_TRACKERS_BY": "Ordina tracker per...",
    "POSITIVE_FIRST": "Prima positivi",
    "NEGATIVE_FIRST": "Prima negativi",
    "ALPHABETICALLY": "A-Z",
    "FAST_SORT": "Ordina rapidamente",
    "SHARE": "Condividi",
    "LOGIN_SUCCESSFUL": "Connesso!",
    "PROTECT_YOUR_DATA": "I dati sono salvati SOLO sul tuo dispositivo!",
    "RESTORE_FROM_BACKUP": "Ripristina da backup",
    "BACKUP_EVERY_24HRS": "Backup ogni 24h",
    "BACKUP_TO_DROPBOX_DESCRIPTION": "Abilita backup su Dropbox per assicurarti di non perdere mai i tuoi dati.",
    "ALL_MY_TRACKERS": "Tutti i miei tracker",
    "NOT_FOR_MEDICAL_ADVICE": "<strong>Nomie è solamente a scopo informativo</strong>. Consultare un medico per informazioni su specifiche condizioni mediche.",
    "HAVE_A_SETUP_CODE": "Hai un Codice Provider?",
    "SETUP_CODE": "Codice Provider",
    "PREBUILT_BOARDS": "Categorie precostruite",
    "NOMIE_AS_IS": "Nomie è fornito \"COSÌ COM'È\", senza garanzia di alcun tipo, esplicita o implicita. Sei pienamente responsabile dei tuoi dati e del loro uso.",
    "BEGIN_TRACKING": "Comincia a tracciare...",
    "CUSTOM_POSITIVITY": "Positività personalizzata", 
    "ONBOARD_WELCOME_MESSAGE": "<strong>Ciao! Benvenuto in Nomie Pro!</strong>,<br /> Ti aiuterò a tracciare, confrontare e capire quello che vuoi della tua vita... <br><strong>in modo completamente privato</strong>.",
    "WHATS_A_TRACKER": "Cosa è un tracker?",
    "WHATS_A_TRACKER_DESCRIPTION": "Un tasto che premi per prendere nota di un momento della tua vita. I tracker possono essere creati da zero o installati dal Nomie Market.<br /><br /><strong>SUGGERIMENTO:</strong> Tieni premuto un tasto per aver accesso a più opzioni.",
    "DELETE_TRACKER_FROM_NOMIE?": "Cancellare questo tracker e tutti i suoi dati?",
    "ARE_YOU_SURE?": "Sei sicuro?",
    "WHATS_A_BOARD": "Che cos'è una categoria?",
    "WHATS_A_BOARD_DESCRIPTION": "Una categoria è una collezione di tracker usati frequentemente. Per esempio una Categoria Cibo può contenere Acqua Bevuta, Pizza, Birra e Caffè.",
    "LOGIN_TO_NOMIE_CONNECT": "Login in Nomie Connect",
    "LOGOUT_OF_NOMIE_CONNECT": "Logout da Nomie Connect",
    "WHAT_IS_NOMIE_CONNECT?": "Nomie Connect ti permette di contribuire al Nomie Market, abilitare un Nomie Orb pubblico e altro...",
    "REGISTER": "Registrati",
    "REGISTER_FOR_NOMIE_CONNECT": "Registrati su Nomie Connect",
    "NOMIE_CONNECT_DISCLAIMER": "Nomie Connect ti permette di contribuire al Nomie Market, abilitare un Nomie Orb pubblico e altro...",
    "NO_ACCOUNT_REGISTER": "Nessun Account? Registrati!",
    "LOGIN": "Login",
    "CANCEL": "Annulla",
    "TAP_TO_COPY": "Clicca per copiare",
    "NEW_IDENTITY_CREATED": "Nuova identità creata",
    "NOMIE_CONNECT": "Nomie Connect",
    "DROPBOX_BACKUP": "Backup Dropbox",
    "ENTER_CODE": "Inserisci Codice Provider",
    "CSV_EXPORT": "Esporta in CSV",
    "COUCHDB_SYNC": "Sincronizza con CouchDB™",
    "ABOUT": "Info",
    "RESET_IDENTITY": "Reset identità",
    "DESTROY_AND_RESET": "Elimina tutti i dati & e fai Reset di Nomie",
    "NOMIE_ORB_SHORT_DESCRIPTION": "Orb online in tempo reale che mostra la tua positività",
    "HAVE_NOMIE_ACCOUNT_QUESTION": "Hai un Account Nomie?",
    "REMOVE": "Rimuovi",
    "DESTROY": "Elimina",
    "DELETE_EVENT": "Rimuovi Evento",
    "SAVE" : "Salva",
    "MORE" : "Più",
    "LESS" : "Meno",
    "CLOSE" : "Chiudi",
    "EVENTS_TODAY" : "Eventi di oggi",
    "ADD_TO_PAST": "Aggiungi in passato",
    "WRITE_NOTE" : "Scrivi una nota",
    "VIEW_STATS" : "Vedi statistiche",
    "PROMPT_FOR_NOTE" : "Chiedi di inserire una nota",
    "PROMPT_FOR_NOTE_DESCRIPTION" : "Lancia automaticamente una nota",
    "WHEN_INPUT_IS" : "Quando l'input è",
     "ADD_CONDITION" : "Aggiungi condizione",
    "UNIT_OF_MEASURE" : "Unità di misura",
    "REORDER": "Riordina",
    "MENU_TODAY_LABEL": "Oggi",
    "MENU_NOTES_LABEL": "Note",
    "MENU_STORE_LABEL": "Market",
    "MENU_MORE_LABEL": "Più",
    "MENU_TRACK_LABEL": "Track",
    "NOTE_SAVED": "Nota Salvata",
    "SAVING": "Salvataggio in corso...",
    "NO_EVENTS_FOR_THIS_DATE": "Nulla di registrato per questa data",
    "PICK_FROM_MY_TRACKERS": "Dalla mia libreria",
    "CREATE_A_TRACKER": "Crea un tracker",
    "CREATE_A_BOARD": "Crea una categoria",
    "ADD_MY_FIRST_BOARD": "Crea la mia prima categoria",
    "ADD_STARTER_BOARD": "Aggiungi una categoria iniziale",
    "BROWSE_MARKET": "Tracker dalla community",
    "GEO_HIGH_ACCURACY_DESC": "Aumenta la precizione della posizione. Nota: Aumenta il tempo di attesa.",
    "DISABLE_MESSAGES_DESC": "Disabilita notifiche per nuovi messaggi",
    "ORGANIZE_TRACKERS": "Riordina tracker",
    "SHOW_BOARD": "Vai a categoria",
    "EXTRA_BOARDS": "Categoria magica",
    "DELETE_BOARD": "Elimina categoria",
    "DELETE_BOARD_DESCRIPTION": "Eliminare una categoria non elimina i tracker. Puoi sempre ricreare questa categoria più tardi",
    "RENAME_BOARD": "Rinomina categoria",
    "BOARD_OPTIONS": "Opzioni categoria",
    "DONE": "Fatto",
    "REFRESH": "Aggiorna",
    "ADD": "Aggiungi",
    "OPTIONS": "Opzioni",
    "MESSAGES" : "Messaggi",
    "TODAYS_LAST_RECORDED_EVENT": "Ultimo evento",
    "FIRST_TIME": "Prima volta",
    "LAST_TIME": "Ultima volta",
    "SINGLE_DAY_RECORD": "Record per un solo giorno",
    "TIME_START_OF_WEEK": "dom",
    "TIME_FORMAT": "h:mm a",
    "TIME_FORMAT_SHORT": "h:mma",
    "DATE_FORMAT_LONG": "ggg MMM Do AAAA",
    "DATE_FORMAT": "MMM G AAAA",
    "DATE_FORMAT_SHORT": "MMM G YYYY",
    "DATE_TIME_FORMAT_LONG": "ggg MMM Do AAAA h:mm a",
    "DATE_TIME_FORMAT_SHORT": "MM/gg/YYYY h:mma",
    "SELECT_POSITIVITY_TITLE": "Nella tua vita, questo è positivo, negativo o neutro?",
    "TRACKER_LABEL_PLACEHOLDER": "Sonno, Caffè, Umore...",
    "LABEL": "Etichetta",
    "DYNAMIC": "Dinamico",
    "AVERAGE": "Media",
    "SUM_TOTAL": "Somma Totale",
    "MINIMUM": "Minimo",
    "MAXIMUM": "Massimo",
    "CALCULATE_TOTAL_BY": "Calcola totale per",
    "NO_NOTES_FOUND": "Nessuna nota trovata",
    "VERY_POSITIVE": "Molto positivo",
    "LAST": "Ultimo",
    "BALANCE": "Bilancio",
    "ICON_AND_COLOR": "Icona e Colore",
    "POSITIVITY": "Positività",
    "POSITIVE": "Positivo",
    "NEUTRAL": "Neutro",
    "NEGATIVE": "Negativo",
    "VERY_NEGATIVE": "Molto negativo",
    "BOARD_NOW": "Ora",
    "BOARD_NOW_DESCRIPTION": "Tracker usati in questo momento",
    "NO_NOW_TRACKERS_MESSAGE" : "Nessun tracker comune trovato per questo momento.",
    "BOARD_POPULAR": "Popolare",
    "BOARD_POPULAR_DESCRIPTION": "Tracker più usati",
    "NO_POPULAR_TRACKERS_MESSAGE" : "Tieni traccia più di frequente per abilitare i tracker popolari",
    "NOTES_TITLE": "Note",
    "SETTINGS" : "Impostazioni",
    "FLAVOR" : "Tema",
    "BUTTON_SIZE" : "Dimensione pulsante",
    "SHOW_TODAY_CHART" : "Mostra elenco di oggi",
    "SHOW_TAB_LABELS" : "Mostra etichette schede",
    "GEO_LOCATION" : "Geolocalizzazione",
    "DISABLE_MESSAGES" : "Messaggi disabilitati",
    "USER_INTERFACE" : "Interfaccia utente",
    "SMALL" : "Piccolo",
    "MEDIUM" : "Medio",
    "LARGE" : "Grande",
    "SELECT" : "Seleziona",
    "GENERATE_NEW_KEYS" : "Genera Nuove Chiavi",
    "INSTALL_AFTERMARKET_APP" : "Installa App Aftermarket",
    "Disable_Auto_Run" : "Metti in Pausa Auto Running App",
    "Disable_Auto_Run_Description" : "Auto-run disabilitato"
}
```

**Comment by Jackymancs4** [comment]
Hello @brandoncorbin
I think this issue can be closed after 65c86fbd35bb878638f359dd856954bcbc6f6834

**Comment by brandoncorbin** [comment]
Closing! Thanks @Jackymancs4
